src/app/learn/accessibility/test/index.html

```diff
@@ -15,7 +15,7 @@
   </stache-page-anchor>
   
   <p>
-    SKY UX incorporates <a href="https://github.com/dequelabs/axe-core">aXe automated tests</a> into its <a routerLink="/learn/get-started/advanced/accessibility-analyzer">accessibility analyzer</a> for use in end-to-end (e2e) tests. aXe includes 50-60 rules covering a subset of the W3C's WCAG 2.0 success criteria. Include accessibility checks in e2e tests using the  <a routerLink="/learn/get-started/advanced/accessibility-analyzer">accessibility analyzer</a>, and ensure that all the rules pass.
+    The <a stacheRouterLink="https://github.com/blackbaud/skyux-lib-testing">SKY UX testing library</a> provides methods to run <a stacheRouterLink="https://github.com/dequelabs/axe-core">aXe automated accessibility tests</a> in your <a stacheRouterLink="/learn/get-started/advanced/accessibility-unit-tests">unit tests</a>. aXe includes 50-60 rules covering a subset of the W3C's WCAG 2.0 success criteria. Include accessibility checks for e2e tests using the  <a routerLink="/learn/get-started/advanced/accessibility-analyzer">accessibility analyzer</a>, and ensure that all the rules pass.
```

Comment:
I hope I'm not being too nitpicky, but it feels off to me to switch context in the last sentence from unit tests to e2e tests. I'd suggest closing the first paragraph with a callout the unit test instructions and moving the reference to accessibility analyzer to the second paragraph. (This would mean making the second paragraph a regular paragraph instead of an includes file.)

For the first paragraph, I'd close with something like: "For guidance on how to run accessibility checks in SKY UX unit tests, see accessibility tests." where "accessibility tests is a link to https://developer.blackbaud.com/skyux/learn/get-started/advanced/accessibility-unit-tests.

For the second paragraph, I'd use a regular paragraph instead of the includes reference to incorporate the a modified reference to accessibility analyzer as a new first sentence and delete the last sentence: "You can also use the accessibility analyzer to include accessibility checks for end-to-end tests. However, SKY UX recommends using unit tests instead of end-to-end tests based on Angular’s guidance on testing. Use e2e tests only as a last resort when you can't use Angular's testing framework to interact with the DOM."

Hopefully this all makes sense, but let me know if not.

Reply:
Not nitpicky! I think it's clearer with those changes so thank you. :) I implemented them and updated the link for the testing library

Reply:
Thanks for making the changes. ... I piggybacked a few additional unrelated changes to headers in the same section because I noticed we weren't consistently following headline capitalization styles. I hope you don't mind.

Reply:
Not at all! Thank you :)